src/System.Runtime.Intrinsics.X86/src/System/Runtime/Intrinsics/X86/Lzcnt.cs

```diff
@@ -0,0 +1,27 @@
+// =+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
+//
+// Lzcnt.cs
```

Comment:
Nit: Should these file headers be removed and replaced with a `<Summary/>` doc comments on the class? e.g.:

```c#
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

using System;
using System.Runtime.Intrinsics;

namespace System.Runtime.Intrinsics.X86
{
    /// <summary>
    /// A class that implements intrinsic functions to provide access to Intel LZCNT instructions.
    /// </summary>
    public static class Lzcnt
    {
        public static bool IsSupported { get; }

        // unsigned int _lzcnt_u32 (unsigned int a)
        public static uint LeadingZeroCount(uint value) { throw new NotImplementedException(); }
        // unsigned __int64 _lzcnt_u64 (unsigned __int64 a)
        public static ulong LeadingZeroCount(ulong value) { throw new NotImplementedException(); }
    }
}
```